monitor/test_result.py

```diff
@@ -90,35 +105,38 @@ def description_display_name(description: str, default_description: str = '') ->
 
 class DbtTestResult(TestResult):
     def __init__(
-        self,
-        id,
-        model_unique_id,
-        test_unique_id,
-        detected_at,
-        database_name,
-        schema_name,
-        table_name,
-        column_name,
-        test_type,
-        test_sub_type,
-        test_results_description,
-        owners,
-        tags,
-        test_results_query,
-        test_rows_sample,
-        other,
-        test_name,
-        test_params,
-        severity,
-        status
+            self,
+            id,
+            model_unique_id,
+            test_unique_id,
+            detected_at,
+            database_name,
+            schema_name,
+            table_name,
+            column_name,
+            test_type,
+            test_sub_type,
+            test_results_description,
+            owners,
+            tags,
+            test_results_query,
+            test_rows_sample,
+            other,
+            test_name,
+            test_params,
+            severity,
+            status
     ) -> None:
         super().__init__(id, model_unique_id, test_unique_id, status)
         self.test_unique_id = test_unique_id
         self.test_type = test_type
         self.database_name = database_name
         self.schema_name = schema_name
         self.table_name = table_name
-        self.table_full_name = '.'.join([database_name, schema_name, table_name]).lower()
+        table_full_name_parts = [database_name, schema_name]
```

Comment:
NIT: I think it should be a method that is called in the init.
IMO init should be straight forward and simple as possible.
Adding if statements inside of it (that are not inline if statements) can make it a bit messy.

Reply:
On one hand, I definitely agreed, that was my instinct as well.
On the other, I figured it's better to follow the pattern (there are already if statements within the constructor).
Either make all of them external functions, or keep all of the logic within the constructor, but I think it's even worse to mix the two. What do you think?

Reply:
I think its better to start the change some where (even if not improving everything) instead of making the original "problem" bigger.
But, this is my opinion and this is a NIT, so I am fine with leaving it like this for now